lib/amplia-client.js

```diff
@@ -232,6 +234,29 @@ class AmpliaClient {
 				.catch((err) => reject(err));
 		});
 	}
+	/**
+	 * Issues a Certificate order from Amplia
+	 *
+	 * @param {string} orderId the ID of the desired order
+	 * @param {string} password the password for the certificate
+	 * @param {number} keySize the size of the key for this certificate
+	 * @returns {IssueSoftwareCertificateResponse} the response containing the certificate in many formats
```

Comment:
Should be "* @returns {Promise\<IssueSoftwareCertificateResponse\>} the response..." here